### PR TITLE
Trouble shooting doc

### PR DESCRIPTION
**Pull request by GuuBu**
Add troubleshooting.md for switch option and create dump. Update README.md and add a new Trouble Shooting section and link to it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,3 +213,6 @@ After this, the code directory should look like C:\php-sdk\phpdev\vc15\x64\php-s
 	$db->close();
   }
 ```
+
+## Trouble shooting
```
Comment (ajlam) [inline README.md]: Combine into one word "troubleshooting"

File: troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Trouble shooting
```
Comment (ajlam) [inline troubleshooting.md]: one word "troubleshooting"

File: troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Possible FAQs
+1.  How to make sure my web page with this extension use redirection when connect to Azure MySQL?
```
Comment (ajlam) [inline troubleshooting.md]: How do I ensure my application uses redirection with this extension to connect to Azure Database for MySQL?

File: troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Trouble shooting
+
+## Possible FAQs
+1.  How to make sure my web page with this extension use redirection when connect to Azure MySQL?  
+
+Answer:  Set mysqlnd_azure.enableRedirect=on   The option value "on" will enforce redirection and return error if the configuration is wrong, e.g. the connection is not configured with SSL.
+
+2. How to verify the connection is currently using redirection?
```
Comment (ajlam) [inline troubleshooting.md:8]: format "host_info" and "PDO::ATTR_CONNECTION_STATUS" as code

**Review by hzhang87 — APPROVED** [review]

**Pull request merged**